fivox/fieldFunctor.h

```diff
@@ -50,40 +50,44 @@ FieldFunctor< TImage >::operator()( const TPoint& point, const TSpacing& ) const
     if( !Super::_source )
         return 0;
 
-    Vector3f base;
-    const size_t components = std::min( point.Size(), 3u );
-    for( size_t i = 0; i < components; ++i )
-        base[i] = point[i];
-
     const float cutOffDistance = Super::_source->getCutOffDistance();
-    const AABBf region( base - Vector3f( cutOffDistance ),
-                        base + Vector3f( cutOffDistance ));
-    const Events& events = Super::_source->findEvents( region );
 
-    const float squaredCutoff = cutOffDistance * cutOffDistance;
-    float sum = 0;
-    for( const Event& event : events )
+    const size_t size = Super::_source->getNumEvents();
```

Comment:
move parts of the comment from lfpFunctor to here to explain this __restrict__, vector aligned magic here?